doc/user/content/manage/disaster-recovery/_index.md

```diff
@@ -0,0 +1,146 @@
+---
+title: "Disaster recovery"
+description: "Learn about various disaster recovery (DR) strategies for Materialize."
+disable_list: true
+menu:
+  main:
+    parent: "manage"
+    weight: 45
+    identifier: "disaster-recovery"
+---
+
+The following outlines various disaster recovery (DR) strategies for
+Materialize.
+
+## Stable Environment ID
+
+For disaster recovery, you must explicitly set the
+`environmentId` in your Materialize CR (Custom Resource). Otherwise, Materialize
+generates a new value each time the environment is recreated, resulting in:
+
+- A new persist namespace in your blob storage
+- Inaccessibility of existing persisted data
+- Full data rehydration from sources
+
+## Level 1: Basic configuration (Intra-Region Recovery)
+
+Because Materialize is deterministic, we recommend running on a container
+scheduler (such as AWS EKS, GCP GKE, Azure AKS). With this setup, the basic
+configuration provides intra-region disaster recovery as long as:
+
+- The scheduler is able to launch a new pod (or equivalent workload) within the
+  region, and
+
+- The underlying object storage service remains available.
+
+In such cases, your mean time to recovery is the **same as your compute
+cluster's rehydration time**.
+
+{{< annotation type="💡 Recommendation" >}}
+
+When running with the basic configuration, we recommend that you track
+your rehydration time to ensure that it is within an acceptable range for your
+business' risk tolerance.
+{{</ annotation >}}
+
+## Level 2:  Multi-replica clusters (High availability across AZs)
+
+Materialize supports multi-replica clusters. To enable distribution of replicas
+across Availability Zones (AZs), specify `cluster_topology_spread_min_domains`
+value greater than 1; e.g.,
```

Comment:
We need to fact check this, what happens if you set it to 3 but only have 1 az?

Reply:
> We need to fact check this, what happens if you set it to 3 but only have 1 az?

Nothing good @sjwiesman, don't do that. 

> added a blurb to set the number to at most # of available zones in your region.
^this is right. It should be set to the number of AZs  (or whatever you topology spread constraints is) that have nodes that can host mz cluster pods.